packages/jest-cli/src/jest.js

```diff
@@ -310,10 +311,58 @@ const runCLI = (
         let testWatcher;
         let timer;
         let displayHelp = true;
+        let typeaheadLinesToClear = 0;
+
+        const typeahead = (hasteMap, patternInfo) => {
+          return new SearchSource(hasteMap, config)
+          .getTestPaths(patternInfo)
+          .then(data => {
+            const {paths} = data;
+            const total = paths.length;
+
+            if (!currentPattern) {
+              return [];
+            } else if (total === 0) {
+              return ['No matching files...'];
```

Comment:
Can you find of a good way to highlight this? Maybe yellow? Check out what happens when you do `jest j23lk2j34l2k` on the repo, it will print a nice "No tests found" message, maybe we could align these two things.